2024/debugging-tips-and-tricks/index.html

```diff
@@ -78,22 +103,30 @@ <h1 style="text-align: left; font-size: 60px">
               mostly work with Widgets and Web Components. And I am presenting
               with my colleague Noah Sager.
             </p>
-            <p>(Noah introduces himself)</p>
             <p>
-              (Max) Today we will provide you with tips and tricks for
-              developing and debugging the apps you build.
+              (Noah) Hi everyone! I am Noah Sager. I am a Product Engineer on
+              the JavaScript team. I work on several major widgets, including
+              Printing, Directions and Measurement.
```

Comment:
@NoashX can you please review & improve the speaker notes on different slides? and this part in particular